cmd/ui-errors.go

```diff
@@ -0,0 +1,170 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cmd
+
+var (
+	uiErrInvalidConfig = newUIErrFn(
+		"Invalid value found in the configuration file",
+		"Please ensure a valid value in the configuration file, for more details refer https://docs.minio.io/docs/minio-server-configuration-guide",
+		"",
+	)
+
+	uiErrInvalidBrowserValue = newUIErrFn(
+		"Invalid browser value",
+		"Please check the passed value",
+		"Browser can only accept `on` and `off` values. To disable web browser access, set this value to `off`",
+	)
+
+	uiErrInvalidCacheDrivesValue = newUIErrFn(
+		"Invalid cache drive value",
+		"Please check the passed value",
+		"MINIO_CACHE_DRIVES: List of mounted drives or directories delimited by `;`.",
+	)
+
+	uiErrInvalidCacheExcludesValue = newUIErrFn(
+		"Invalid cache excludes value",
+		"Please check the passed value",
+		"MINIO_CACHE_EXCLUDE: List of cache exclusion patterns delimited by `;`.",
+	)
+
+	uiErrInvalidCacheExpiryValue = newUIErrFn(
+		"Invalid cache expiry value",
+		"Please check the passed value",
+		"MINIO_CACHE_EXPIRY: Cache expiry duration in days.",
+	)
+
+	uiErrInvalidCredentials = newUIErrFn(
+		"Passed credentials are not suitable for use",
+		"Please provide correct credentials",
+		`Access key length should be between minimum 3 characters in length.
+Secret key should not be between 8 and 40 characters.`,
+	)
+
+	uiErrInvalidErasureEndpoints = newUIErrFn(
+		"Invalid endpoints to use in erasure mode",
+		"Please provide correct combinations of local/remote paths",
+		"For more information, please refer to the following link: https://docs.minio.io/docs/minio-erasure-code-quickstart-guide",
+	)
+
+	uiErrInvalidNumberOfErasureEndpoints = newUIErrFn(
+		"The total number of endpoints is not suitable for erasure mode",
+		"Please provide an even number of endpoints greater or equal to 4",
+		"For more information, please refer to the following link: https://docs.minio.io/docs/minio-erasure-code-quickstart-guide",
+	)
+
+	uiErrStorageClassValue = newUIErrFn(
+		"Invalid storage class value",
+		"Please check the passed value",
+		`MINIO_STORAGE_CLASS_STANDARD: Format "EC:<Default_Parity_Standard_Class>" (e.g. "EC:3"). This sets the number of parity disks for Minio server in Standard mode. Object are stored in Standard mode, if storage class is not defined in Put request.
+MINIO_STORAGE_CLASS_RRS: Format "EC:<Default_Parity_Reduced_Redundancy_Class>" (e.g. "EC:3"). This sets the number of parity disks for Minio server in reduced redundancy mode. Objects are stored in Reduced Redundancy mode, if Put request specifies RRS storage class.
```

Comment:
Please make capitalization of mode names consistent. 
Use either lower case or upper case. IMHO, upper case is better.
`reduced redundancy mode`
=>
`Reduced Redundancy mode`